### PR TITLE
feat: Complete fb

### PR DESCRIPTION
**Pull request by plll0123**
solve #1

### CODE REVIEW

File: dofb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for i in range(1, 20):
```
Comment (pll0123) [inline dofb]: it's not pythonic

File: dofb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    if i%15==0:
```
Comment (pll0123) [inline dofb]: Do fizzbuzz with 1 if statement

**Review by pll0123 — CHANGES_REQUESTED** [review]
DO THIS RIGHT AWAY!!!

File: dofb (hunk 1 above)
Comment (plll0123) [reply dofb]: yes..

File: dofb (hunk 2 above)
Comment (plll0123) [reply dofb]: I'll do it

**Review by pll0123 — APPROVED** [review]
아니.. 할 줄 아시면서 ..

**Pull request merged**